crates/bevy_input/src/gamepad.rs

```diff
@@ -154,7 +154,14 @@ impl ButtonSettings {
     }
 }
 
-/// Stores settings for axis.
+/// Defines the sensitivity range and threshold for an axis.
+///
+/// Values that are lower `negative_high` will be rounded to -1.0.
+/// Values that are higher than `positive_high` will be rounded to 1.0.
+/// Values that are in-between `negative_low` and `positive_low` will be rounded to 0.0.
+/// Otherwise, values will not be rounded.
+///
+/// `threshold` defines the minimum difference between old and new values to apply the changes.
```

Comment:
Maybe someone else has other opinions, but personally I would put this on the `threshold` field itself.

You could maybe even move the other field descriptions to their respective properties as well.

Reply:
I moved that comment to the `threshold` field.
I think it would be better to leave other comments on the top. If you have other opinion, share you suggestion